implementations/contracts/ERC725/IERC725X.sol

```diff
@@ -48,5 +48,5 @@ interface IERC725X {
         address to,
         uint256 value,
         bytes calldata data
-    ) external payable;
+    ) external payable returns(bytes memory);
```

Comment:
Could you also update the ERC725 docs adding the bytes array and return values? Maybe first make the or to this repo before we do at the eip